packages/pieces/community/google-slides/src/lib/actions/generate-from-template.ts

```diff
@@ -0,0 +1,120 @@
+import { googleSlidesAuth } from '../../index';
+import { createAction, DynamicPropsValue, Property } from "@activepieces/pieces-framework";
+import { getSlide, PageElement, batchUpdate } from '../commons/common';
+import { google } from 'googleapis';
+import { OAuth2Client } from 'googleapis-common';
+
+export const generateFromTemplate = createAction({
+    name: 'generate_from_template',
+    displayName: 'Generate from template',
+    description: 'Generate a new slide from a template',
+    auth: googleSlidesAuth,
+    props: {
+        template_presentation_id: Property.ShortText({
```

Comment:
Don't you also need the target directory ID to move the copy to the right location?